crates/biome_cli/src/reporter/terminal.rs

```diff
@@ -210,10 +222,21 @@ impl fmt::Display for SummaryTotal<'_> {
 pub(crate) struct ConsoleTraversalSummary<'a>(
     pub(crate) &'a TraversalMode,
     pub(crate) &'a TraversalSummary,
+    pub(crate) bool,
 );
 impl fmt::Display for ConsoleTraversalSummary<'_> {
     fn fmt(&self, fmt: &mut Formatter) -> io::Result<()> {
-        let summary = SummaryTotal(self.0, self.1.changed + self.1.unchanged, &self.1.duration);
+        let mut duration = self.1.duration;
+        if !self.2 {
+            if let Some(scanner_duration) = self.1.scanner_duration {
+                duration += scanner_duration;
+            }
+        } else if let Some(scanner_duration) = self.1.scanner_duration {
+            let scanned = ScanSummary(&scanner_duration);
+            fmt.write_markup(markup!(<Info>{scanned}</Info>))?;
+            fmt.write_str("\n")?;
+        }
+        let summary = SummaryTotal(self.0, self.1.changed + self.1.unchanged, &duration);
```

Comment:
Please destructure the tuple here, so the items get named:

```suggestion
        let (mode, summary, verbose) = self;
        let mut duration = summary.duration;
        if !verbose {
            if let Some(scanner_duration) = summary.scanner_duration {
                duration += scanner_duration;
            }
        } else if let Some(scanner_duration) = summary.scanner_duration {
            let scanned = ScanSummary(&scanner_duration);
            fmt.write_markup(markup!(<Info>{scanned}</Info>))?;
            fmt.write_str("\n")?;
        }
        let summary = SummaryTotal(mode, summary.changed + summary.unchanged, &duration);
```